packages/client/src/v2-events/features/debug/debug.tsx

```diff
@@ -10,12 +10,46 @@
  */
 /* stylelint-disable */
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools'
-import React from 'react'
+import { onlineManager } from '@tanstack/react-query'
+
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+
+const wasOffline = localStorage.getItem('debugger-offline') === 'true'
```

Comment:
<img width="1088" alt="image" src="https://github.com/user-attachments/assets/8bf7415e-a047-4230-be7b-6c20563d9002" />

Setting the Tanstack debugger offline now persist making development of offline functionality a lot easier